src/OpenTK/Platform/MacOS/QuartzDisplayDeviceDriver.cs

```diff
@@ -121,14 +116,27 @@ public QuartzDisplayDeviceDriver()
                             opentk_dev_current_res = thisRes;
                         }
                     }
+                    //Copy the current display mode and take a pointer to it
+                    IntPtr displayMode = CG.CopyDisplayMode(currentDisplay);
+                    //Pull out the scaled width and height
+                    int scaledWidth = (int)CG.GetModeWidth(displayMode);
+                    int scaledHeight = (int)CG.GetModeHeight(displayMode);
+                    //Pull out the raw width and height
+                    int rawWidth = (int)CG.GetModePixelWidth(displayMode);
+                    int rawHeight = (int)CG.GetModePixelHeight(displayMode);
+                    //Remember to release the display mode
+                    CG.ReleaseDisplayMode(displayMode);
+
+                    float pixelScaleW = (float)rawWidth / scaledWidth;
+                    float pixelScaleH = (float)rawHeight / scaledHeight;
```

Comment:
`pixelScaleW` and `pixelScaleH` are never used in this code? I'm guessing this is something that never got removed?